### ISSUE TITLE
ONNX runtime unit tests failed in Jenkins...

### ISSUE THREAD

**Issue by scxiao** [ROCmSoftwarePlatform/AMDMIGraphX#743]
There is a regression that some ONNXruntime unit tests failed. So we disabled those tests with the following change:
```
- ./build.sh --config Release --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --test --use_migraphx
+ ./build.sh --config Release --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --use_migraphx
```
For now, we cannot reproduce the failure locally in a docker container, but we need to investigate why these tests failed in Jenkins.

**Comment by mvermeulen** [comment]
Try with ROCm 4.2 and/or ROCm 4.3 RC1

**Comment by scxiao** [comment]
@mvermeulen @pfultz2 The develop branch failed with this PR merged, and the error is the same as before, so the problem is still there. I am wondering whether we can revert this PR out from the develop branch or create another PR to  turn off those tests?

**Comment by causten** [comment]
Fixed by  ROCm 4.2
